lib/index.js

```diff
@@ -205,21 +205,21 @@ exports.compile = function(str, options){
   var parsed = parse(str, options);
   if (options.compileDebug !== false) {
     fn = [
-        'var jade_debug = [ new jade.DebugItem( 1, ' + filename + ' ) ];'
+        'var pug_debug = [ new pug.DebugItem( 1, ' + filename + ' ) ];'
       , 'try {'
       , parsed.body
       , '} catch (err) {'
-      , '  jade.rethrow(err, jade_debug[0].filename, jade_debug[0].lineno' + (options.compileDebug === true ? ',' + utils.stringify(str) : '') + ');'
+      , '  pug.rethrow(err, pug_debug[0].filename, pug_debug[0].lineno' + (options.compileDebug === true ? ',' + utils.stringify(str) : '') + ');'
       , '}'
     ].join('\n');
   } else {
     fn = parsed.body;
   }
-  fn = new Function('locals, jade', fn)
+  fn = new Function('locals, pug', fn)
```

Comment:
They were completely un-documented, so I think it's OK with this still being a potential breakage when moving from `jade@1.x.x` to `pug@1.x.x`.  To be honest though, it's not something I have a strong opinion on.  Mostly I'm far more interested in getting people moved over to pug.

We only have 3 issues left before a stable release.  I'll try and look at some fo them over the weekend.